src/frontend/src/sass/_buttons.scss

```diff
@@ -89,7 +89,8 @@ a:hover {
   }
 }
 
-.btn-login {
+.btn-login,
+%btn-login {
```

Comment:
I was receiving the same error that @binggaofw was getting when extending using '.btn-login' so simply added the placeholder version to be identically defined for use below.